clicommand/artifact_search.go

```diff
@@ -165,7 +170,11 @@ Format specifiers:
 		}
 
 		if len(artifacts) == 0 {
-			l.Fatal(fmt.Sprintf("No matches found for %q", cfg.Query))
+			if cfg.AllowEmptyResults {
+				l.Info(fmt.Sprintf("No matches found for %q", cfg.Query))
+			} else {
+				l.Fatal(fmt.Sprintf("No matches found for %q", cfg.Query))
```

Comment:
this isn't your fault as it's inspired by pre-existing code, but our logger functions automatically Sprintf-ify things:
```suggestion
			if cfg.AllowEmptyResults {
				l.Info("No matches found for %q", cfg.Query)
			} else {
				l.Fatal("No matches found for %q", cfg.Query)
```